src/pages/iou/steps/NewRequestAmountPage.tsx

```diff
@@ -180,14 +168,12 @@ function NewRequestAmountPage({route, iou, report, selectedTab}) {
     );
 }
 
-NewRequestAmountPage.propTypes = propTypes;
-NewRequestAmountPage.defaultProps = defaultProps;
 NewRequestAmountPage.displayName = 'NewRequestAmountPage';
 
-export default withOnyx({
+export default withOnyx<NewRequestAmountPageProps, NewRequestAmountPageOnyxProps>({
     iou: {key: ONYXKEYS.IOU},
     report: {
-        key: ({route}) => `${ONYXKEYS.COLLECTION.REPORT}${lodashGet(route, 'params.reportID', '')}`,
+        key: ({route}) => `${ONYXKEYS.COLLECTION.REPORT}${route.params.reportID}`,
```

Comment:
No '' default value, should we add it here?

Reply:
Let's avoid defaulting to `''`, [it can produce unexpected results](https://github.com/Expensify/react-native-onyx?tab=readme-ov-file#dependent-onyx-keys-and-withonyx).